ksqldb-engine/src/main/java/io/confluent/ksql/util/ScalablePushQueryMetadata.java

```diff
@@ -139,8 +159,33 @@ public ResultType getResultType() {
 
   public void onException(final Consumer<Throwable> consumer) {
     runningFuture.exceptionally(t -> {
+      scalablePushQueryMetrics.ifPresent(metrics ->
```

Comment:
Ah, ok.  I see, I think we do the same thing in pull queries.  So if an error occurs while running, after the initial start, this is what counts it as an error, right?